base/applications/mspaint/winproc.cpp

```diff
@@ -668,10 +673,19 @@ LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
             imageModel.Redo();
             break;
         case IDM_EDITCOPY:
+            // FIXME: We should use CF_DIB in the future
```

Comment:
Do you mean, CF_DIB *in addition* to CF_BITMAP? Or just CF_DIB? (some apps, maybe MS Paint too (but to be verified), save data in clipboard in more than 1 format)
Or, will copying in CF_DIB automatically perform conversion to CF_BITMAP and make that other format also available in clipboard?